### ISSUE TITLE
Players are still having issues with growing BoP flowers via the config settings

### ISSUE THREAD

**Issue by Sunconure11** [GrowthcraftCE/Growthcraft-1.7#213]
Not sure yet if this was fixed in the latest build, but, it appears in build 3 (latest build you pushed out before merging), players still cannot grow BoP flowers if I set them to grow, at least, in my modpack. The only flower that seems to grow... is hibiscus. Testing in the latest build after the merge still yields this issue.

In case its a config fault at this point, here is my file:

http://pastebin.com/xz5HnGia

Note: Briefly had the wildcard set to -1, but, I changed it.

https://www.dropbox.com/s/wc3m3aeetu437ns/Screenshot%202016-02-22%2020.39.04.png?dl=0

Hibiscuses appear to be growing, but, other flowers are not.

**Comment by IceDragon200** [comment]
@Sunconure11 https://github.com/GrowthcraftCE/Growthcraft-1.7/wiki/User-API-(2.5.x)#flowers-entries

> **Note** Flower entries have a entry_type field, this is used to control the type of Flower block registered, whether it be "generic" (uses canPlaceBlockAt to determine if the flower can be placed), OR "forced", in which the block goes through common block placement checks, but never relies on the canPlaceBlockAt behaviour, use this entry_type for flowers from BiomesOPlenty or other mods where the flowers cannot be placed under normal circumstances. Please use "forced" with caution, as it can cause odd behavior with some flowers.

tl;dr change the "entry_type" to "forced"

``` json
{
  "comment": "",
  "data": [
    {
      "comment": "",
      "entry_type": "generic",
      "block": {
        "comment": "tile.flower2.name",
        "mod_id": "minecraft",
        "name": "red_flower",
        "meta": 32767
      }
    },
    {
      "comment": "",
      "entry_type": "generic",
      "block": {
        "comment": "tile.flower1.name",
        "mod_id": "minecraft",
        "name": "yellow_flower",
        "meta": 32767
      }
    },
    {
      "comment": "BiomesOPlenty flowers require a forced entry, in order for it to be placed by the bee box spawning.",
      "entry_type": "forced",
      "block": {
        "comment": "",
        "mod_id": "BiomesOPlenty",
        "name": "flowers",
        "meta": 32767
      }
    },
    {
      "comment": "BiomesOPlenty flowers require a forced entry, in order for it to be placed by the bee box spawning.",
      "entry_type": "forced",
      "block": {
        "comment": "",
        "mod_id": "BiomesOPlenty",
        "name": "flowers2",
        "meta": 32767
      }
    }
  ]
}
```

This would have been generated in the default config by the way with the latest builds

**Comment by Sunconure11** [comment]
Thanks, I'll try this out. I'll close it for now unless this does nothing.

Edit: Issue is fixed, though updating appears to have caused another weird issue.
